tools/check_release.py

```diff
@@ -376,15 +419,19 @@ def get_latest_library_versions(repo_path):
     logging.basicConfig(level=level)
     
     # Read configuration data
-    json_data = json.load(open(os.path.join(os.path.dirname(__file__), "check_release.json")))
-    
+    json_data = json.load(open(os.path.join(os.path.dirname(__file__), 
+                               "check_release.json")))
```

Comment:
Ahhhh!!! `data`?! you want to call it DATA! 

Also, open files that are leaked (not really gced language) are closed when gced. This is generally pretty safe, but the timing of closing the file descriptor can be wonky.